packages/flutter/lib/src/material/drawer_header.dart

```diff
@@ -55,6 +56,9 @@ class DrawerHeader extends StatelessWidget {
   /// If the child is null, the padding has no effect.
   final EdgeInsets padding;
 
+  /// The margin used to surround the child.
```

Comment:
This should discuss how it's distinct from `padding` (which has a very similar description).